### DISCUSSION TITLE
Short cut for hiding done/checked things. [enhancement]

### DISCUSSION THREAD

**Discussion by aldaw** [vslinko/obsidian-outliner#343] [Ideas]
A short cut to hide done/checked elements would be nice.
I am also willing to pay for adding this feature.
Do you have also a PayPal addy where someone can directly make a donation?

**Comment by vslinko** [comment]
@aldaw thank you for the feedback!

What do you mean by hiding done/checked elements? Remove them from a list? Could you provide more information about your use case.

I had a PayPal for donations, but I removed it because I can't guarantee that I'll provide consistent time.
